packages/backend/lambda/functions/hello.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+func handler(request events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+	return &events.APIGatewayProxyResponse{
+		StatusCode: 200,
```

Comment:
Best practice as of: https://httpstatuses.com/200 is to import the htto module and use `http.StatusOK`